tooling/release_download_test.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I'm not sure we always have bash on our systems @sxa ?

Reply:
various scripts in the tooling directory use bash, so I assumed that bash would be fine.
The main reason to use bash it to be able to specify `set -o pipefail` which I consider to be a must for complex shell scripts imho.

Reply:
@karianna I had this discussion with @netomi elsewhere too as I'd prefer not to use bash if possible (IMHO using bash constructs leads to less readable scripts ;-) ) but we do have bash on all our systems for other things, and certainly on the systems which the jenkins job that runs this will be targetted at.

Reply:
I can also revert to use /bin/sh but then I have to remove the set -o pipefail.